Solutions/AristaAwakeSecurity/Analytic Rules/HighMatchCountsByDevice.yaml

```diff
@@ -0,0 +1,64 @@
+id: 90b7ac11-dd6c-4ba1-a99b-737061873859
+name: Awake Security - High Match Counts By Device
+description: This query searches for devices with unexpectedly large number of activity match.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
+      - AwakeSecurity
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Arista Networks" and DeviceProduct == "Awake Security"
+  | summarize Models=make_set(Activity), ASPMatchURLs=make_set(DeviceCustomString2), SourceIPs=make_set(SourceIP),
```

Comment:
Wondering if we also wanted to bring the TimeGenerated field something like | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated)

Reply:
Added to rules